### ISSUE TITLE
Feature: Prisma Client Observer

### ISSUE THREAD

**Issue by jeremygiberson** [FormidableLabs/envelop-network-viewer#4]
Prisma is a popular ORM in the Node ecosystem. Providing a prisma observer would allow observing database queries for all of the following databases:

- PostgreSQL
- MySQL
- SQLite
- MongoDB
- CockroachDB
- Microsoft SQL Server

We can make use of the Prisma middleware to observe queries sent to the databases. https://www.prisma.io/docs/concepts/components/prisma-client/middleware

The observations added to the log statement might look like:
```json
{
  "label": "Prisma",
  "data": {
    "queryCount": 2,
    "queries": [
      {
        "model": "Author",
        "action": "create",
        "duration_ms": "2"
      },
      {
        "model": "Author",
        "action": "update",
        "args": { "where": "id=3", "orderBy": ["name asc"]},
        "duration_ms": "3"
      }
    ]
  }
}
```

Usage of this observer would be opt-in. Initialization might look like:
```javascript
const prisma = new PrismaClient()

const useNetworkViewerConfig = {
    additionalObservers: [new PrismaObserver(prisma, {/* observer config*/})]
};

const getEnveloped = envelop({
  plugins: [
    useNetworkViewer(true, useNetworkViewerConfig)
  ]
})
```

**Comment by jeremygiberson** [comment]
Available as of 1.3.0

**Comment by jeremygiberson** [comment]
Available as of 1.3.0
